lib/versioning.h

```diff
@@ -152,6 +152,7 @@
 /* config version code, in the same format as GlobalConfig->version */
 #define VERSION_VALUE   0x0319
 #define VERSION_CURRENT_VER_ONLY "3.25"
+#define VERSION_LAST_SEMANTIC_CHANGE VERSION_3_22
```

Comment:
This value has to be manually maintained every time we change the **config version** (i.e. new `cfg_is_config_version_older` call), which has the risk that we forget it.

Until we find out something better, please add a one-line comment above it suggesting that this has to be the same as the latest version value used in all `cfg_is_config_version_older` calls.